modular_skyrat/modules/SEVA_suit/code/seva_obj.dm

```diff
@@ -11,7 +11,7 @@
 	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
 	w_class = WEIGHT_CLASS_BULKY
-	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
```

Comment:
instead of a number you should probably use one of the defines for this, like: FIRE_SUIT_MAX_TEMP_PROTECT

Reply:
(seems to work for the storm-immunity deal aswell) 
https://github.com/Skyrat-SS13/Skyrat-tg/blob/7df83aea5542ef40c803e54474ed4711f7f92407/code/datums/weather/weather_types/ash_storm.dm#L69

Reply:
> instead of a number you should probably use one of the defines for this, like: FIRE_SUIT_MAX_TEMP_PROTECT

the only define there was for that is at the 35000 K one

Reply:
> > instead of a number you should probably use one of the defines for this, like: FIRE_SUIT_MAX_TEMP_PROTECT
> 
> the only define there was for that is at the 35000 K one

true;

would be wise to modularize storm protection (should be like, 1 line) in lieu of breaking the suits 1(singular) purpose

Reply:
> > > instead of a number you should probably use one of the defines for this, like: FIRE_SUIT_MAX_TEMP_PROTECT
> > 
> > 
> > the only define there was for that is at the 35000 K one
> 
> true;
> 
> would be wise to modularize that part of storm protection (should be like, 1-2 lines) in lieu of breaking the suits 1(singular) purpose

lemme see if we can make a global specific to it - I know there's a trait for it